includes/functions.php

```diff
@@ -34,12 +34,46 @@
  *
```

Comment:
Docblock says `Fetch stories based on customizer settings.`

But this  isn't really related to the customizer it seems?

Reply:
This function does use customizer, if the function called without any arguements it will take the Customizer settings and render the stories.

Reply:
Separated functions.